controllers/transactions.controller.js

```diff
@@ -60,11 +60,30 @@ module.exports = {
   }),
   getAllTransactions: catchAsync(async (req, res, next) => {
     try {
-      const response = await Transactions.findAll();
+      let { page=0 } = req.query
+
+      const datos = {
+        next:0,
+        previous:0,
+        aux:page,
+        aux2:page,
+        offset:page
+      }
+      page = +page
+      if(page>0){
+        datos.previous = --datos.aux2
+        page+=page
+      }
+      datos.offset = datos.offset*10
+      datos.next = ++datos.aux;
+      const response = await Transactions.findAll({offset:datos.offset,limit:10});
+      
       const idQuery = req.query.userId;
       if (idQuery) {
         const responseId = await Transactions.findAll({
           where: { userId: `${idQuery}` },
+          offset:datos.offset,
+          limit:10
```

Comment:
add pagination when passed the userId:

![Captura de Pantalla 2022-11-12 a la(s) 12 33 55 p m](https://user-images.githubusercontent.com/84211273/201487090-a8195169-39eb-4bcc-baa5-2ded5dd1be83.png)